src/plugins/foxPage/components/AssetActions.tsx

```diff
@@ -11,30 +11,23 @@ import {
   Tabs,
   Text as CText,
 } from '@chakra-ui/react'
-import { generateOnRampURL } from '@coinbase/cbpay-js'
-import { adapters, AssetId, fromAccountId } from '@shapeshiftoss/caip'
+import { AssetId } from '@shapeshiftoss/caip'
 import { foxyAddresses } from '@shapeshiftoss/investor-foxy'
-import { getConfig } from 'config'
 import { FoxyPath } from 'features/defi/providers/foxy/components/FoxyManager/FoxyCommon'
 import qs from 'qs'
-import { useEffect, useState } from 'react'
 import { useTranslate } from 'react-polyglot'
 import { useHistory, useLocation } from 'react-router'
-import coinbaseLogo from 'assets/coinbase-pay/cb-pay-icon.png'
```

Comment:
We can also remove the `src/assets/coinbase-pay` folder from the repo

Reply:
actually this one needs to stay

Reply:
oh you're right, killed